app/models/spotlight/resources/oaipmh_mods_parser.rb

```diff
@@ -189,5 +193,55 @@ def transform_to_iiif_thumbnail(ids_uri)
       uri = uri.gsub(/\/full\/\d*,\d*\/0\/default.jpg/, '')
       uri += "/full/#{thumbnail_size},/0/native.jpg"
     end
+
+    # Spotlight v3.3.0
+    # Spotlight expects "exhibit-specific fields" (a.k.a. Exhibit#custom_fields) to not have
+    # a Solr suffix (e.g. _tesim, _ssim, etc.). This method assumes all non-configured fields
+    # are custom and thus removes their Solr suffix when adding them to the @item_sidecar hash.
+    # Configured fields are added as-is (Solr suffix included).
+    def assign_item_sidecar_data(field_name, value)
```

Comment:
Refer to #3 under the "Long version" section of this PR's description for an explanation